textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/comparablematcher/ComparableMatcherTest.java

```diff
@@ -50,6 +50,7 @@ public void setUp() throws Exception {
         for (ITuple tuple : TestConstants.getSamplePeopleTuples()) {
             dataWriter.insertTuple(tuple);
         }
+        ((DataWriter) dataWriter).close();
```

Comment:
@qinhuah Since @zuozhiw changed `IDataWriter` to `IDataWriter` in his commit https://github.com/TextDB/textdb/pull/328/commits/1fe0d18b23880f1294835db176fdbed0fcd9a003 , please make the same change here.  As a consequence, there is no need to do type casting.  Make the same change to other test cases.